src/Components/Map/FilterMap.js

```diff
@@ -220,7 +220,7 @@ function FilterMap(props) {
               </Row>
 
               {/* NAME SELECT ////////////////////////////////////////////////////////////////// */}
-              <Row className="mb-1">
+              {/* <Row className="mb-1">
```

Comment:
Instead of leaving this commented out, please delete this chunk of code entirely